src/trainx.py

```diff
@@ -263,6 +421,14 @@ def train(mcfg):
     if os.path.isfile(args.cfg_file):
         mc = get_model_config(args.cfg_file)
 
+    if tf.gfile.Exists(mc.train.LOG_TRAIN_DIR):
```

Comment:
check for existence and delete + makedirs should be in a function and not copy pasted